pkg/codegen/testing/test/testdata/azure-native-nested-types/nodejs/tsconfig.json

```diff
@@ -10,13 +11,19 @@
         "experimentalDecorators": true,
         "noFallthroughCasesInSwitch": true,
         "forceConsistentCasingInFileNames": true,
+		"paths": {
+            "@/*": ["*"]
+		},
```

Comment:
Nit: It looks like this is using tabs rather than spaces for the indentation. Mind fixing it to consistently use spaces?

Reply:
Damn, I thought I fixed that. Thanks Justin!